pytest.ini

```diff
@@ -0,0 +1,46 @@
+[pytest]
+# https://docs.pytest.org/en/stable/reference.html#confval-addopts
```

Comment:
There's no need to reference generic option docs unless. This is only useful when you use unconventional settings.
```suggestion
```